lib/modules/versioning/semver-coerced/index.ts

```diff
@@ -77,9 +96,10 @@ function isLessThanRange(version: string, range: string): boolean {
 function isGreaterThan(version: string, other: string): boolean {
   const coercedVersion = semver.coerce(version);
   const coercedOther = semver.coerce(other);
-  return coercedVersion && coercedOther
-    ? semver.gt(coercedVersion, coercedOther)
-    : false;
+  if (!coercedVersion || !coercedOther) {
+    return false;
+  }
+  return semver.gt(coercedVersion, coercedOther);
```

Comment:
why is this changed?

Reply:
nothing but to make the `return false;` more clear, I don't mind reverting it if you wish so.